src/DataSource.ts

```diff
@@ -12,12 +12,18 @@ import { GithubDataSourceOptions, GitHubQuery, GitHubVariableQuery, Label } from
 import { replaceVariables } from './variables';
 import { isValid } from './validation';
 import { getAnnotationsFromFrame } from 'common/annotationsFromDataFrame';
+import { prepareAnnotation } from 'migrations';
+import QueryEditor from 'views/QueryEditor';
 
-export class DataSource extends DataSourceWithBackend<GitHubQuery, GithubDataSourceOptions> {
+export class GithubDataSource extends DataSourceWithBackend<GitHubQuery, GithubDataSourceOptions> {
   templateSrv = getTemplateSrv();
 
   constructor(instanceSettings: DataSourceInstanceSettings<GithubDataSourceOptions>) {
     super(instanceSettings);
+    this.annotations = {
+      QueryEditor,
```

Comment:
[nit] There no need to specify `QueryEditor` if you are just using default QueryEditor.